component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -828,6 +819,27 @@ public Response updateUserById(@Context UriInfo uriInfo,
     return EntityBuilder.getResponse(EntityBuilder.buildEntityProfile(id, uriInfo.getPath(), expand), uriInfo, RestUtils.getJsonMediaType(), Response.Status.OK);
   }
 
+  @GET
+  @Path("onBoarding/{id}")
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("administrators")
+  @ApiOperation(value = "Send onBoarding email to a specific user", 
+                httpMethod = "GET", 
+                response = Response.class, 
```

Comment:
not sure GET is the right verbal fsemantic for an endpoint that actually changes stats. I would use POST here to  follow RESTful recommends

Reply:
:+1:

Reply:
I think the most correct is to use PATCH , as we used when update profile property in updateUserProfileAttribute